.rspec

```diff
@@ -1,2 +1,3 @@
 --color
+--format progress
```

Comment:
Since the order specs are run in isn't fixed, might be worth considering using `--format documentation`, so one could see the order at a glance (and debug issues if order happens to be a factor in flakiness). Thats my 2 cents.

Reply:
knapsack_pro in Regular Mode shows at the begining of output something like 

```
rspec  --default-path spec "spec/system/articles/user_visits_articles_by_timeframe_spec.rb" ...
```

You can copy paste to run in the same order tests.

In Queue Mode there will be more examples in output for each given set of tests fetched from Queue. Thanks to that you can run tests in development in the same order.

If you use RSpec seed then you can pass additionally --seed to retry in exact order https://knapsackpro.com/faq/question/how-to-find-seed-in-rspec-output-when-i-use-queue-mode-for-rspec

Reply:
cool, I see it now, thanks! no `--format documentation` needed

Reply:
one day I think it'd be nice to use `--order random` but I don't think we're quite there yet/that could probably be a separate piece of work to get working